specs/deneb/polynomial-commitments.md

```diff
@@ -162,33 +158,33 @@ def multi_exp(points: Sequence[TPoint],
 #### `hash_to_bls_field`
 
 ```python
-def hash_to_bls_field(data: bytes) -> BLSFieldElement:
+def hash_to_bls_field(data: bytes) -> bls.Scalar:
```

Comment:
Perhaps add some notes above here or the "## Introduction" section to describe `bls.Scalar` shortly?

Reply:
No longer necessary.